taker-frontend/src/components/Trade.tsx

```diff
@@ -239,6 +243,19 @@ export default function Trade({
                                 >
                                     Long
                                 </Button>
+                                <Button
+                                    // canSubmitShort is not a problem
+                                    disabled={!canSubmitShort}
+                                    colorScheme="red"
+                                    size="lg"
+                                    // using onShortOpen fails
```

Comment:
what are these comments? are there TODOs?

Reply:
oh no they can be removed. i put them there to document an issue i was having which is now resolved